api/router.go

```diff
@@ -103,5 +103,29 @@ func NewRouter(s *Server) *mux.Router {
 	r.Methods("DELETE").Path("/v1/engineimages/{name}").Handler(f(schemas, s.EngineImageDelete))
 	r.Methods("POST").Path("/v1/engineimages").Handler(f(schemas, s.EngineImageCreate))
 
+	settingListWatcher := s.wsc.NewWatcher("setting")
+	defer settingListWatcher.Close()
```

Comment:
Don't quite understand how this works. Are watchers closed immediately after the route has been set?